generators/component/templates/component.ts

```diff
@@ -46,6 +88,11 @@ export class <%= component.class %> extends<% if (component.isExtendingMes) { %>
 
     //#endregion
 
+    /**
+     * Constructor
+     * 
+     * @param viewContainerRef the reference to the component view container
+     */
     constructor(viewContainerRef: ng.ViewContainerRef) { 
```

Comment:
@jpsfs @pesimeao Different concerns gentlemen. If it does not give you the elementRef to use in the progress indicator, either inject it yourself or add it here as well.
ViewContainerRef is a preparation for CMFFLex use as it has more leverage than ElementRef.